diff -Nur zlib-1.2.8-/configure zlib-1.2.8/configure
--- zlib-1.2.8-/configure	2015-08-03 20:49:44.000000000 +0500
+++ zlib-1.2.8/configure	2015-08-03 20:50:52.000000000 +0500
@@ -165,7 +165,7 @@
 if test "$gcc" -eq 1 && ($cc -c $test.c) >> configure.log 2>&1; then
   echo ... using gcc >> configure.log
   CC="$cc"
-  CFLAGS="${CFLAGS--O3} ${ARCHS}"
+  CFLAGS="${CFLAGS--O3} -fPIC ${ARCHS}"
   SFLAGS="${CFLAGS--O3} -fPIC"
   LDFLAGS="${LDFLAGS} ${ARCHS}"
   if test $build64 -eq 1; then
